### PR TITLE
Update picker UX

### PR DESCRIPTION
**Pull request by donnapep**
Update picker UX and clean up repo.
[Screen shot](https://cloud.githubusercontent.com/assets/1190420/14832028/2a29aed0-0bc6-11e6-8021-cebc61bf8381.png)

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees @rodrigopavezi Please review. Thx.

File: package.json [hunk 1]
```diff
@@ -18,30 +18,19 @@
...
-    "gulp": "~3.8.1",
+    "gulp": "~3.8.3",
     "gulp-bump": "~0.1.8",
     "gulp-concat": "^2.2.0",
-    "gulp-html2js": "^0.1.0",
+    "gulp-html2js": "^0.2.0",
     "gulp-jshint": "~1.5.5",
-    "gulp-protractor": "~0.0.8",
     "gulp-rename": "~1.2.0",
     "gulp-rimraf": "^0.1.0",
```
Comment (stulees) [inline package.json]: @donnapep Can you replace `gulp-rimraf` with `del`

**Comment by stulees** [comment]
@donnapep One suggestion, otherwise 👍

**Comment by donnapep** [comment]
@stulees Replaced it. Thx.

**Pull request merged**